### ISSUE TITLE
Add the capability to open Image Viewer immediately while dragging timed-Images to plot

### ISSUE THREAD

**Issue by helenayele** [debrief/debrief#3355]
Currently while dragging timed-images to plot, nothing is shown, or the images are not opened in image viewer view. But it helps a lot for a user of  debrief adds this capability just like it did to Media Player view.
### Steps to Reproduce:
 

1. Open Sample.dpf file
2. Drag TimedImages to the plot from other_formats folder
3. See that nothing is shown.
4. Open Image Viewer view
5. Drag the image to the view , this time images with proper naming are loaded to the view.

### Expected Result
When we drag timed images , Image viewer should be opened and load the images properly.

**Comment by IanMayo** [comment]
Yes, @helenayele - I agree. This would be a useful capability.  It would also match the existing behaviour of media player.
